packages/vscode-extension/src/common/LaunchConfig.ts

```diff
@@ -25,6 +25,7 @@ export type LaunchConfigurationOptions = {
     buildType?: string;
     productFlavor?: string;
   };
+  packageManager?: string;
```

Comment:
Enum too?

Reply:
we are not sure about what the user has put there that check comes latter